src/components/Charts/StackedAreaChart.jsx

```diff
@@ -128,10 +127,11 @@ export const mapStateToProps = state => {
     to: state.query.date_received_max
   };
   const interval = state.query.dateInterval;
+  // clone the data so this doesn't mutate redux store
   const processData = cloneDeep( data )
   const filteredData = pruneIncompleteStackedAreaInterval(
-      processData, dateRange, interval );
-  const showChart = filteredData.length > 2
+    processData, dateRange, interval );
+  const showChart = filteredData.length > 1
```

Comment:
This is correct. Number of points for both charts should match.

For both charts, the number of points you need to actually see something meaningful should be at least 2.  We could put that in a constant so we can reference it somewhere like const MIN_POINTS = 2;